rust/core-lib/src/editor.rs

```diff
@@ -621,12 +609,12 @@ impl Editor {
         // Another possibility would be to make the delta builder be able to handle
         // overlapping deletions (using union semantics).
         let mut deletions = Selection::new();
-        for r in self.view.sel_regions() {
+        for &r in self.view.sel_regions() {
```

Comment:
Since we're iterating over `&T` and the type is `Copy`, we can pattern-match it as `&r` so `r` becomes `T`. Without `Copy` it would give an error about trying to move from a borrowed context.